landing-pages/site/static/.htaccess

```diff
@@ -1,9 +1,11 @@
+RedirectMatch Permanent ^/((1\.10\.[0-9]+|2\.[0-9.]+|stable)/.*)$ "https://airflow.apache.org/docs/apache-airflow/$1"
+RedirectMatch Permanent ^/docs/((1\.10\.[0-9]+|2\.[0-9.]+|stable)/.*)$ "https://airflow.apache.org/docs/apache-airflow/$1"
+
+# Airflow 1.10.x
 RedirectMatch Permanent ^/docs/(stable|1.10.10)/api(\.html)?$ "https://airflow.apache.org/docs/apache-airflow/$1/rest-api-ref"
 RedirectMatch Permanent ^/docs/(stable|1.10.10)/(cli|macros)(\.html)?$ "https://airflow.apache.org/docs/apache-airflow/$1/$2-ref"
```

Comment:
These won't match anymore, as lines 1 and 2 take over. So I think these need to go first.

Reply:
okay, I reverted the order.